Arquillian tests fails when running from Intellij Idea

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

When running any test from arquillian testsuite with Intellij idea, it fails as those tests are using embedded undertow by default. And Keycloak fails to start with Embedded undertow since the commit https://github.com/keycloak/keycloak/commit/25186278fceb79de296cd8d232dbf82ea6aa1de0 .

### Version

main (after commit https://github.com/keycloak/keycloak/commit/25186278fceb79de296cd8d232dbf82ea6aa1de0 )

### Regression

- [x] The issue is a regression

### Expected behavior

Keycloak is possible to test with embedded undertow.

### Actual behavior

Keycloak is not possible to test with embedded undertow.

### How to Reproduce?

For example:
```
mvn clean install -DskipTests=true -Pdistribution
cd testsuite/integration-arquillian/tests/base/
mvn clean install -Dtest=LoginTest
```

It fails with the exception like:
```
Caused by: java.lang.NullPointerException: Cannot invoke "String.isEmpty()" because "v" is null
	at org.keycloak.utils.JsonConfigProvider$JsonScope.toString(JsonConfigProvider.java:93)
	at java.base/java.util.Optional.map(Optional.java:260)
	at org.keycloak.utils.JsonConfigProvider$JsonScope.get(JsonConfigProvider.java:88)
	at org.keycloak.Config$AbstractScope.getValue(Config.java:241)
	at org.keycloak.Config$AbstractScope.getBoolean(Config.java:237)
	at org.keycloak.services.DefaultKeycloakSessionFactory.isEnabled(DefaultKeycloakSessionFactory.java:367)
	at org.keycloak.services.DefaultKeycloakSessionFactory.loadFactories(DefaultKeycloakSessionFactory.java:335)
	at org.keycloak.services.DefaultKeycloakSessionFactory.init(DefaultKeycloakSessionFactory.java:102)
	at org.keycloak.services.resteasy.ResteasyKeycloakApplication.createSessionFactory(ResteasyKeycloakApplication.java:82)
	at org.keycloak.services.resources.KeycloakApplication.startup(KeycloakApplication.java:76)
	at org.keycloak.testsuite.TestPlatform.onStartup(TestPlatform.java:50)
	at org.keycloak.services.resources.KeycloakApplication.<init>(KeycloakApplication.java:65)
	... 76 more

```

### Anything else?

_No response_

--- a/services/src/test/java/org/keycloak/utils/JsonConfigProvider.java
+++ b/services/src/test/java/org/keycloak/utils/JsonConfigProvider.java
@@ -90,7 +90,7 @@ public class JsonConfigProvider implements Config.ConfigProvider {
 
         private String toString(JsonNode n) {
             String v = replaceProperties(n.textValue());
-            return !v.isEmpty() ? v : null;
+            return !StringUtil.isNullOrEmpty(v) ? v : null;
         }
 
         @Override

